.github/workflows/partial-test-build.yaml

```diff
@@ -43,18 +39,27 @@ on:
         required: false
         type: boolean
         default: true
-      distPath:
-        description: The path to the dist folder
+      source_path:
+        description: The path to the folder or file to deploy
```

Comment:
I'm in doubt about this change. For devs `source` is a `src` folder where we have our source code before it's compiled.
`dist` - is alway compiled code that should be deployed.